R/FilterStates.R

```diff
@@ -367,12 +367,13 @@ FilterStates <- R6::R6Class( # nolint
     #'
     ui = function(id) {
       ns <- NS(id)
-      private$cards_container_id <- ns("cards")
+      private$cards_container_id <- ns("filter-card-container")
```

Comment:
`filter-card-container` uses `-` which is a shiny namespace separator. I'm not sure whether it has consequences or risks.

Reply:
Yeah I'd change that to "_"